acl_upgrade.php

```diff
@@ -480,19 +482,67 @@
   updateAcl($doc_write, 'Physicians', 'patients', 'Patients', 'amendment', 'Amendments (write,addonly optional)', 'write');
   //Insert the 'lab' object from the 'patients' section into the Physicians group write ACL (added in 5.0.1)
   updateAcl($doc_write, 'Physicians', 'patients', 'Patients', 'lab', 'Lab Results (write,addonly optional)', 'write');
-
+  //Insert the 'Multipledb' object from the 'admin' section into the Administrators group write ACL (added in 5.0.1)
+  updateAcl($admin_write, 'Administrators','admin', 'Administration', 'Multipledb', 'multipledb','write');
   //DONE with upgrading to this version
   $acl_version = $upgrade_acl;
 }
 
-/* This is a template for a new revision, when needed
+ //This is a template for a new revision, when needed
 // Upgrade for acl_version 5
 $upgrade_acl = 5;
 if ($acl_version < $upgrade_acl) {
   echo "<B>UPGRADING ACCESS CONTROLS TO VERSION ".$upgrade_acl.":</B></BR>";
 
   //Collect the ACL ID numbers.
   echo "<B>Checking to ensure all the proper ACL(access control list) are present:</B></BR>";
+//Get Accountant ACL ID number
+    $admin_write = getAclIdNumber('Administrators', 'write');
+
+
+  //Add new object Sections
+  echo "<BR/><B>Adding new object sections</B><BR/>";
+    // Add 'Groups (write,view)' object (added in 5.0.1)
+    addObjectSectionAcl('Groups', 'groups');
+
+
+ //Add new Objects
+  echo "<BR/><B>Adding new objects</B><BR/>";
+
+    addObjectAcl('admin', 'Administration', 'gadd'  , 'Add/Update groups');
+    addObjectAcl('admin', 'Administration', 'gcalendar'  , 'Create/Update groups appointment in calender');
+    addObjectAcl('admin', 'Administration', 'glog'  , 'Group log');
+    addObjectAcl('admin', 'Administration', 'gdlog'  , 'Group detailed log of appointment in patient record');
+    addObjectAcl('admin', 'Administration', 'gm'  , 'Send message from the permanent group therapist to the personal therapist');
```

Comment:
acl_upgrade.php
 -----------------
![image](https://cloud.githubusercontent.com/assets/18376627/24835307/98ed1a20-1d07-11e7-9089-fb7b5dd9e8bf.png)



acl_setup.php
-------------------

![image](https://cloud.githubusercontent.com/assets/18376627/24835304/8037abc6-1d07-11e7-8ea0-0c5ca31d8b31.png)

they seem to be the same